### PR TITLE
Currency : Adding build_script for py4j:0.10.9

### PR DESCRIPTION
**Pull request by shubham-garud**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: p/py4j/py4j_0.10.9_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+mvn install
+mvn test
+
+cd ..
+cd py4j-python/
+pip3 install -r requirements-test.txt
+
+#Tests taking lot of time for execution. Please uncomment if needed.
+#pytest
+
```
Comment (asowani) [inline p/py4j/py4j_0.10.9_ubi_8.5.sh:47]: @shubham-garud - An addition of a "cd ../.." command would be desirable here, though it is not crucial. The reason: once the script completes, user will be returned to the original directory from which the script was started.

Comment (shubham-garud) [reply p/py4j/py4j_0.10.9_ubi_8.5.sh:47]: Thanks for letting me know this @asowani  . Will update it.

**Review by asowani — APPROVED** [review]
Reviewed suggested changes.

**Comment by seth-priya** [comment]
@shubham-garud since you have commented the test execution, can you attach the logs to this issue?

**Comment by seth-priya** [comment]
@nikhil-kalbande are we okay with this from a currency perspective?

**Comment by nikhil-kalbande** [comment]
> @nikhil-kalbande are we okay with this (skipping the tests) from a currency perspective?

@seth-priya  we have limitation on travis for build time. But for Currency Jenkins builds, we can accommodate build/test taking extra time.
@shubham-garud  if tests are passing we should keep those enabled in build script. If travis build fails for you, attach local execution logs and we will merge the PR.

**Comment by shubham-garud** [comment]
**Testcases passing successfully locally but travis check failing due to time elapsed and network error.**

pytest -k "not java_tls_test. and not java_gateway_test."
====================== test session starts ====================================
platform linux -- Python 3.8.13, pytest-7.2.2, pluggy-1.0.0
rootdir: /py4j
collected 189 items / 77 deselected / 112 selected

py4j-python/src/py4j/tests/byte_string_test.py .                                                                                                      [  0%]
py4j-python/src/py4j/tests/client_server_test.py .................                                                                                    [ 16%]
py4j-python/src/py4j/tests/finalizer_test.py .......                                                                                                  [ 22%]
py4j-python/src/py4j/tests/java_array_test.py .......                                                                                                 [ 28%]
py4j-python/src/py4j/tests/java_callback_test.py .......................                                                                              [ 49%]
py4j-python/src/py4j/tests/java_dir_test.py .........                                                                                                 [ 57%]
py4j-python/src/py4j/tests/java_help_test.py .........                                                                                                [ 65%]
py4j-python/src/py4j/tests/java_list_test.py ............                                                                                             [ 75%]
py4j-python/src/py4j/tests/java_map_test.py ..                                                                                                        [ 77%]
py4j-python/src/py4j/tests/java_set_test.py ...                                                                                                       [ 80%]
py4j-python/src/py4j/tests/memory_leak_test.py ..............                                                                                         [ 92%]
py4j-python/src/py4j/tests/py4j_signals_test.py ...                                                                                                   [ 95%]
py4j-python/src/py4j/tests/signals_test.py .....                                                                                                      [100%]

==================================== warnings summary ===========================
py4j-python/src/py4j/tests/java_gateway_test.py:192
  /py4j/py4j-python/src/py4j/tests/java_gateway_test.py:192: PytestCollectionWarning: cannot collect test class 'TestConnection' because it has a __init__ constructor (from: py4j-python/src/py4j/tests/java_gateway_test.py)
    class TestConnection(object):

py4j-python/src/py4j/tests/client_server_test.py::GarbageCollectionTest::testSendObjects
  /py4j/py4j-python/src/py4j/tests/client_server_test.py:148: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(1000, hello.calls)

py4j-python/src/py4j/tests/client_server_test.py::IntegrationTest::testMultiClientServer
  /py4j/py4j-python/src/py4j/tests/client_server_test.py:612: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(thisThreadId,

py4j-python/src/py4j/tests/client_server_test.py::IntegrationTest::testMultiClientServer
  /py4j/py4j-python/src/py4j/tests/client_server_test.py:614: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(thisThreadId,

py4j-python/src/py4j/tests/client_server_test.py::IntegrationTest::testMultiClientServerWithSharedJavaThread
  /py4j/py4j-python/src/py4j/tests/client_server_test.py:569: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(sharedPythonThreadId0,
 
py4j-python/src/py4j/tests/client_server_test.py::IntegrationTest::testMultiClientServerWithSharedJavaThread
  /py4j/py4j-python/src/py4j/tests/client_server_test.py:571: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(sharedPythonThreadId1,

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
==============112 passed, 77 deselected, 6 warnings in 103.87s (0:01:43) ====================

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**